deployment/united-manufacturing-hub/templates/kafkatopostgresqlv2/deployment.yaml

```diff
@@ -86,6 +86,10 @@ spec:
               value: {{.Values.kafkatopostgresqlv2.asset.cache.lru.size | quote}}
             - name: VALUE_CHANNEL_SIZE
               value: {{.Values.kafkatopostgresqlv2.workers.channel.size | quote}}
+            - name: VALUE_TABLE_SIZE
```

Comment:
Can we use a default value if this is not set yet? simplifies the upgrade process :)

- name: FLUSH_INTERVAL_SECONDS
  value: {{ .Values.kafkatopostgresqlv2.postgresql.flush.interval | default "10" | quote }}

Reply:
yes, can do